src/requirements.yml

```diff
@@ -5,6 +5,10 @@
   name: amazon_ssm_agent
 - src: https://github.com/cisagov/ansible-role-assessment-tool
   name: assessment_tool
+- src: https://github.com/cisagov/ansible-role-automated-security-updates
+  name: automated_security_updates
+- src: https://github.com/cisagov/ansible-role-backports
+  name: backports
```

Comment:
I believe this can be removed here since backports is not used in this repo [anymore](https://github.com/cisagov/terraformer-packer/commit/3a686d90ad74a4e3a21fd93d3d3fda5d6ad0d32d).
```suggestion
```